src/ontology/cl-edit.owl

```diff
@@ -3070,6 +3070,7 @@ Declaration(Class(obo:CP_0000043))
 Declaration(Class(obo:D96882F1-8709-49AB-BCA9-772A67EA6C33))
 Declaration(Class(obo:GO_0005903))
 Declaration(Class(obo:GO_0031045))
+Declaration(Class(obo:GO_0036053))
```

Comment:
Can you also add this term in the GO imports txt file? I will refresh the imports later.